docs/Browser Support.md

```diff
@@ -0,0 +1,73 @@
+# Browser Support
+
+<!-- @import "[TOC]" {cmd="toc" depthFrom=2 depthTo=6 orderedList=false} -->
+
+<!-- code_chunk_output -->
+
+- [Guidelines](#guidelines)
+- [Polyfills](#polyfills)
+
+<!-- /code_chunk_output -->
+
+VTEX browser support is split in two contexts: `storefront` and `admin areas`.
+
+For `storefronts`, we strive to support all major browsers all the way to IE 11:
+
+```text
+# Browserslist example (date 29/01/2020)
+IE 11
+Edge 17+
+Safari 8+
+Chrome 77+
+Firefox 70+
+```
+
+For `admin areas`, the main targets are evergreen browsers, such as Firefox, Chrome, the new Edge, etc:
+
+```text
+# Browserslist example (date 29/01/2020)
```

Comment:
Nothing about IE11? Wondering If we could check already with analytics the use of the admin by store owners.

Reply:
That would be great indeed! Let's move this to the slack channel